### ISSUE TITLE
Imported LESS file is not processed.

### ISSUE THREAD

**Issue by smlombardi** [livestyle/issues#74]
Most of our sites have a main styles.less/styles.scss file which is processed into the css file linked i the html.This is working perfectly in LiveStyle.

However, we ALSO have many imported less and scss files in those main files.  Sometime even those main files are nothing but a manifest of imports.

Our imported files are the complete css of the project, split up for convenience and organization.  They contain much more than mixins and variables.

For example, today I tried out LS with a current LESS-based site.  Code in styles.less perfectly updated.  However, editing code in _homepage.less nothing happened.

I realized there's overhead with imports, but with so many projects split into includes like _home.less or _navigation.less I don't know how I'm going to be able to use LiveStyle.

**Comment by sergeche** [comment]
LS workflow implies that, when you edit `_home.less` from your example, you must associate it with compiled stylesheet in browser.

Yes, it’s a bit awkward to re-associate stylesheets when you switch to another file. But I need more time to investigate a better solution and UI for this problem. For example, read dependencies from a top-level preprocessor stylesheet and automatically map all editor files that `@import`’ed into it to compiled counterpart.

**Comment by yurri87** [comment]
я не могу понять как подключить .sass файл
подскажите пожалуйста!

I can not connect sass file.
help me please!

**Comment by sergeche** [comment]
SASS-файлы не поддерживаются, только SCSS

**Comment by yurri87** [comment]
даже так. У меня в ST3 открыт style.scss, но в плагине для хрома он не отображен. Я не могу понять как его подключить.

**Comment by sergeche** [comment]
У файла стоит синтаксис SCSS?

**Comment by yurri87** [comment]
нет, стоял CSS, видимо в этом была проблема, но я не вижу в списке синтаксисов SCSS...

**Comment by sergeche** [comment]
Значит, надо установить пакет с синтаксисом SCSS. Это легко делается через Package Control

**Comment by yurri87** [comment]
это помогло!
Спасибо огромное Сергей!
С Новым Годом!

**Comment by sergeche** [comment]
Отлично :+1: Спасибо, и вас тоже!
